__tests__/commands/FQL10Functions.test.js

```diff
@@ -73,6 +73,10 @@ describe("FQL 10 Functions", () => {
     await cleanup();
   });
 
+  afterAll(async () => {
+    faunaClient.close();
+  });
```

Comment:
what does this do? it looks resonable, I'm just curious if it was fixing a bug related to other changes, or if theres some other reason this is here.

Reply:
It leaves jest with open handles, so jest doesn't terminate cleanly :/